pkg/observability/components/Dashboard/InstallCrdView.vue

```diff
@@ -19,12 +19,10 @@ export default {
         await this.$store.dispatch('observability/setMissingCrd', false);
       } catch (err) {
         handleGrowl(this.$store, {
-          error: {
-            message: `${ this.t('observability.errorMsg.failedCrd') } ${
-              err.message ? `: ${ err.message }` : ''
-            }`,
-            type: 'error',
-          },
+          message: `${ this.t('observability.errorMsg.failedCrd') } ${
```

Comment:
did you test all of these object changes to the messages for the growls? did they work fine (did the growl display as expected)?

Reply:
no, I didn't, but I changed the signature of the function without changing the behavior, so if it did work then it will work after